libkbfs/errors.go

```diff
@@ -1193,3 +1193,15 @@ type MutableBareRootMetadataNoImplError struct {
 func (e MutableBareRootMetadataNoImplError) Error() string {
 	return "Does not implement MutableBareRootMetadata"
 }
+
+// blockNonExistentError is returned when a block doesn't exist.
```

Comment:
Added comment. I just felt icky using server-only errors in things that aren't servers.